src/main/java/com/elytradev/concrete/block/ConcreteBlock.java

```diff
@@ -0,0 +1,181 @@
+package com.elytradev.concrete.block;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+
+import java.util.Optional;
+import java.util.Random;
+import java.util.function.Supplier;
+
+/**
+ * An extension of {@link Block} used by all blocks in Unnamed.
```

Comment:
This javadoc isn't really useful at all, it should be removed or replaced with an explanation of the class, or the entire block builder "module".